README.md

```diff
@@ -111,6 +119,18 @@ The `grunt ts` task is preconfigured with to targets, `dev` and `dist`. The task
 
 Where the `distDirectory` is defaulted to `dist/umd`.
 
+If you have other root level directories besides `src` and `tests`, you might want to exclude them from the `dist` build. Including them can cause unwanted output in your `dist/umd` directory. For example, if you wanted to exclude an `examples` directory, you would add:
+
+```json
```

Comment:
either make this legit JSON (putting `"` on the keys) or mark it as js.

Reply:
converted to `js`!